src/WebApp/wwwroot/lib/co-design/css/codidact.css

```diff
@@ -0,0 +1 @@
ble.table th{border-bottom:3px solid #ccc;padding:6px 10px;font-weight:bold;text-align:left}table.table td{border-bottom:1px solid #ccc;padding:6px 10px;text-align:left}table.table th:not(:last-child),table.table td:not(:last-child){border-right:1px solid #ccc}table.table.is-compact th,table.table.is-compact td{padding:2px 6px}table.table.is-striped tr:nth-of-type(2n+1) td{background-color:#f6f6f6}table.table.is-with-hover tr:hover td{background-color:#eee}img{max-width:100%;height:auto}*{box-sizing:border-box}html,body{margin:0;padding:0}.container{max-width:72rem;margin:0 auto}.grid{display:flex;flex-wrap:wrap;margin:-5px}.grid.is-nowrap{flex-wrap:nowrap}.grid .grid--cell{flex-grow:0;flex-shrink:0;padding:5px}.grid .grid--cell.is-flexible{flex-grow:1;flex-shrink:1}.grid .grid--cell.is-1{width:8.3333333333%}.grid .grid--cell.is-2{width:16.6666666667%}.grid .grid--cell.is-3{width:25%}.grid .grid--cell.is-4{width:33.3333333333%}.grid .grid--cell.is-5{width:41.6666666667%}.grid .grid--cell.is-6{width:50%}.grid .grid--cell.is-7{width:58.3333333333%}.grid .grid--cell.is-8{width:66.6666666667%}.grid .grid--cell.is-9{width:75%}.grid .grid--cell.is-10{width:83.3333333333%}.grid .grid--cell.is-11{width:91.6666666667%}.grid .grid--cell.is-12{width:100%}@media all and (max-width: 32rem){.grid .grid--cell.is-1-sm{width:8.3333333333%}.grid .grid--cell.is-2-sm{width:16.6666666667%}.grid .grid--cell.is-3-sm{width:25%}.grid .grid--cell.is-4-sm{width:33.3333333333%}.grid .grid--cell.is-5-sm{width:41.6666666667%}.grid .grid--cell.is-6-sm{width:50%}.grid .grid--cell.is-7-sm{width:58.3333333333%}.grid .grid--cell.is-8-sm{width:66.6666666667%}.grid .grid--cell.is-9-sm{width:75%}.grid .grid--cell.is-10-sm{width:83.3333333333%}.grid .grid--cell.is-11-sm{width:91.6666666667%}.grid .grid--cell.is-12-sm{width:100%}}@media all and (min-width: 32rem)and (max-width: 64rem){.grid .grid--cell.is-1-md{width:8.3333333333%}.grid .grid--cell.is-2-md{width:16.6666666667%}.grid .grid--cell.is-3-md{width:25%}.grid .grid--cell.is-4-md{width:33.3333333333%}.grid .grid--cell.is-5-md{width:41.6666666667%}.grid .grid--cell.is-6-md{width:50%}.grid .grid--cell.is-7-md{width:58.3333333333%}.grid .grid--cell.is-8-md{width:66.6666666667%}.grid .grid--cell.is-9-md{width:75%}.grid .grid--cell.is-10-md{width:83.3333333333%}.grid .grid--cell.is-11-md{width:91.6666666667%}.grid .grid--cell.is-12-md{width:100%}}@media all and (min-width: 64rem){.grid .grid--cell.is-1-lg{width:8.3333333333%}.grid .grid--cell.is-2-lg{width:16.6666666667%}.grid .grid--cell.is-3-lg{width:25%}.grid .grid--cell.is-4-lg{width:33.3333333333%}.grid .grid--cell.is-5-lg{width:41.6666666667%}.grid .grid--cell.is-6-lg{width:50%}.grid .grid--cell.is-7-lg{width:58.3333333333%}.grid .grid--cell.is-8-lg{width:66.6666666667%}.grid .grid--cell.is-9-lg{width:75%}.grid .grid--cell.is-10-lg{width:83.3333333333%}.grid .grid--cell.is-11-lg{width:91.6666666667%}.grid .grid--cell.is-12-lg{width:100%}}/*# sourceMappingURL=codidact.css.map */
```

Comment:
Another option would be to use the npm package. Example: https://unpkg.com/@codidact/co-design@0.5.0/dist/codidact.css.

Anyway: needs to be updated to version 0.5.0

Reply:
This was discussed back and forth in codidact/core#45 and codidact/core#46. The problem is that Npm puts the packages in `node_modules` but we need it somewhere else. It is also an additional package manager, we already use Nuget for C# stuff. A git submodule is one way of dealing with this relatively painlessly. (We would setup a Github action thingy that automatically creates the appropriate `dist/X.X.X` tag when a new `version/X.X.X` tag is created.)

Reply:
unpkg is a cdn, which means that we don't need to host anything for now.

Reply:
So we would just use something like the following?

~~~html
<link rel=stylesheet href="https://unpkg.com/@codidact/co-design@0.5.0/dist/codidact.css" />
~~~

Is this feasible for production? I mean is that server available 24/7, *et cetera*?

Reply:
I did a quick google search and this seems feasible, let's do this, if there are problems with down times we can still use the submodule stuff.

Reply:
Agreed. /cc @misha130

Reply:
I wish it was npm because I don't want to update the version number here but rather in a package json but this is fine for now.

Anyhow I made a commit where I removed the lib and put in the link to the unpkg